### ISSUE TITLE
Error when loading masks

### ISSUE THREAD

**Issue by RakhithaRR** [matterport/Mask_RCNN#1334]
I'm trying to train the model on my own dataset with 4 classes using the COCO pretrained weights. But before training, I get the error `index ' ' is out of bounds for axis ' ' with size ' '`. There are different values in different instances.

I think the error occurs because the mask values returned from `skimage.draw.polygon` exceed the indices in the array. I can't seem to figure out a way to correct this problem. Any help would be appreciated. I followed the balloons script and coco script when creating the script for my custom dataset.

**Comment by ridwaniyas** [comment]
Yes, in my case, the error occurs because I drew the polygon outside the boundary of the images. The JSON files seems want to understand the coordinate of annotations, but it is out of limit (index '...'). Just try to double check your annotations and the format. If you use balloon script, the format have to be the same with VIA 1.x. Hope this help
